tests/subsys/dfu/dfu_target/mcuboot/src/main.c

```diff
@@ -46,14 +52,24 @@ int dfu_target_mcuboot_write(const void *const buf, size_t len)
 
 int dfu_target_mcuboot_done(bool successful)
 {
-	return done_retval;
+	int ret = done_retval;
+
+	write_retval = -1;
+	offset_get_retval = -1;
+	done_retval = -1;
+	return ret;
 }
 
 int dfu_target_mcuboot_schedule_update(int img_num)
 {
 	return schedule_retval;
 }
 
+int  dfu_target_mcuboot_reset(void)
```

Comment:
```suggestion
int dfu_target_mcuboot_reset(void)
```
Did you clang-format these changes?

Reply:
Thanks, good spotting.. Most of the lines I did run through the formatter, but not full files.